src/app/blogs/layout.tsx

```diff
@@ -1,6 +1,6 @@
-import Link from "next/link";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+
+import BackButton from "./BackButton";
```

Comment:
We may not use backbutton.tsx again, But its good for some future needs... **Kudos**, as a suggestion and more of a best practice, its better to put the button in components under shared file structure to make it reusable

Reply:
Done!